kmod/src/fence.c

```diff
@@ -238,6 +238,11 @@ int scoutfs_fence_start(struct super_block *sb, u64 rid, __be32 ipv4_addr, int r
 	struct pending_fence *fence;
 	int ret;
 
+	if (!rid) {
```

Comment:
Sadly, I don't think this'll work.  Depending on where the fence came from we might not end up cleaning it up.  See the _fence_next calls in the server that lead to _recov_finished.  We'll need to more about the precise source of the fencing request to avoid it.

Reply:
So, we know it comes from `scoutfs_net_reconn_free_worker+0x1bd` and not the other 2 callers. I'll look into that.

Also, we can just stopgap and not create sysfs entries here, too, and just let all the fencing happen. The worst that could happen, I guess, is that we end up fencing the wrong clients that also have rid=0, so maybe not such a big deal?